src/models/product.model.ts

```diff
@@ -0,0 +1,9 @@
+import { Document } from 'mongoose';
+
+export interface Product extends Document {
```

Comment:
verify if it adds id, use id instead of name everywhere

Reply:
yes there is id on Document